packages/@aws-cdk/aws-acmpca/package.json

```diff
@@ -92,7 +92,7 @@
     "node": ">= 10.13.0 <13 || >=13.7.0"
   },
   "stability": "experimental",
-  "maturity": "cfn-only",
```

Comment:
Why?
Looks like it only have an [l1](https://github.com/aws/aws-cdk/tree/master/packages/%40aws-cdk/aws-acmpca)

Reply:
Not sure if this qualifies as an L2 - https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-acmpca/lib/certificate-authority.ts

Reply:
I think the `certificate-authority.ts` definitely makes it an L2.